packages/frontend-shared/cypress/support/e2e.ts

```diff
@@ -360,7 +360,26 @@ function visitApp (href?: string, opts?: Partial<Cypress.VisitOptions>) {
 
       return config.clientRoute
     }).then((clientRoute) => {
-      return cy.visit(`http://localhost:${e2e_serverPort}${clientRoute || '/__/'}#${href || ''}`, opts)
+      if (href) {
+        return cy.visit(`http://localhost:${e2e_serverPort}${clientRoute || '/__/'}#${href || ''}`, opts)
+      }
+
+      cy.visit(`http://localhost:${e2e_serverPort}${clientRoute || '/__/'}`, opts)
+
+      // we know the main route will go to the specs list
+      // just make sure the page is done rendering before taking any action
+      if (appSetup === 'new-project') {
+        // if this is a new project, we'll be on the create spec page
+        return cy.get('[data-cy=create-spec-page-cards]').should('be.visible')
+      }
+
+      if (appSetup === 'no-specs') {
+        // if this is an existing project with no specs, we'll be on the no specs found page
+        return cy.get('[data-cy=create-spec-page-description]').should('be.visible')
+      }
```

Comment:
Is the `spec-list-container` rendered with these project setups as well?

Reply:
Sadly no. I'm leaning towards creating a new custom command to just verify the specs page is present, so going to try that to not overload the visitApp command

Reply:
I'm surprised our UI changes these values so much